### ISSUE TITLE
Edge Chromium crashes when downloading a file. This only happens in Cairo.

### ISSUE THREAD

**Issue by andy0dvlpr** [cairoshell/cairoshell#314]
I use Microsoft Edge Dev (Chromium version) as my preferred browser. The browser worked very well and with no issues at all before I installed Cairo. Afterwards, whenever I wanted to download a file using the browser, it crashed.
I know this is not a problem from Edge Chromium because once I uninstalled Cairo everything worked as expected.

**Steps to reproduce**
1.  Install Edge Chromium
2. Install Cairo
3. Try to download a file

**Expected behavior**
The file should have had downloaded without any problems.

**System information:**
 - Windows version: Windows 10 Version 1909
 - Cairo version: 0.3.6842.0 - Pre-release
 - 32-bit or 64-bit: 64-bit
 - In Cairo Settings > Advanced, is Cairo set as shell?: Yes
- Edge Chromium Version: 81.0.396.0

**Additional context**
I think that the issue might have something to do with the animation that Edge does when downloading a new file. This might be wrong, however.

**Comment by RUvlad1** [comment]
I have the same browser, Windows and Cario version but this does not happen to me. But, I have it so it asks me where the file should be saved. Try turning that on

**Comment by josuave** [comment]
I have the production version of Edge (Chromium) and this does not happen to me. My downloads are auto saved to Downloads.

**Comment by TSCode91** [comment]
Can confirm this..I have the same problem with it..Chromium edge crashes when downloading a file

**Comment by dremin** [comment]
I am unable to reproduce this on Edge 80 and Edge Dev (82). Is it still happening for you @andy0dvlpr and @TSCode91 ?

**Comment by andy0dvlpr** [comment]
> I am unable to reproduce this on Edge 80 and Edge Dev (82). Is it still happening for you @andy0dvlpr and @TSCode91 ?

Sorry for the late response, I have stopped checking my emails for a while. I have reinstalled Cairo today and this issue is no longer happening. I do not know if it is because of the numerous Windows, Edge or Cairo updates that have happened during the time I've been missing but I'm glad to see that this doesn't happen anymore. I will now close the issue.
